### DISCUSSION TITLE
RFQ - Different post date per multisite entry

### DISCUSSION THREAD

**Discussion by lenvanessen** [craftcms/cms#9008] [Ideas]
### Description

We're using "Let each entry choose which site it's active on" propagation setting heavily in our system, and it's great. However, we also schedule a lot of content, and the only downsite with this system is that you can't schedule a different postdate for a translation/other site. Possibly related to #2116. Would be great if this was possible somehow.

**Comment by lenvanessen** [comment]
@brandonkelly don't mean to push, but do you have an idea around the feasibility of implementing this? We're kind of in debating between a couple of paths, both are suboptimal in terms of implementation. But knowing if this can be addressed will help us put it into perspective and plan this on our timeline.

1. Create a translatable postDate field and not use the built in field. However, we run into a heap of problems there, for example:
- Blitz doesn't know when the cache should be invalidated
- There's multiple plugins (SEOmatic etc) that use this field for al heap of things.

2. Work around the problems of having no-propagation between the sites
- Allow for duplication between the sites using for instance the Duplicate Entries extension
- Manually set the SEOmatic settings for alternate links etc
- Override functionality for the Language switcher, getSupportedSites method etc.

**Comment by brandonkelly** [comment]
@lenvanessen This would be a relatively involved change, so not something we’d do outside of a major release (4.0/5.0/etc.), and only if it were to pick up some steam.

**Comment by lenvanessen** [comment]
Additional background:
For more complex publishing platforms, the simple fact that post-dates and authors are not localizable creates many problems:

We cannot schedule content properly, all articles have to go live at the same time, where is usually the translations will have to wait.
If you update a article on site A, by default sorting, it will be listed in the top position on site B, even though it has not been updated there
We can't control Craft's awesome rights functions, because all sites have their own authors, and we can only assign one to an entry that is propagated trough all sites. So features like "only edit their own articles" cannot be used.
By some hack-ish solution we have just created a new table with:
siteId
authorId
postDate

To store these values localized, and we throw them into place in the beforePrepare and afterPrepare of the EntryQuery, but surely there is a nicer approach that can be taken from the Craft Core.
